### ISSUE TITLE
Attributes in STATE are throwing PROP errors/warnings

### ISSUE THREAD

**Issue by evandavis** [yannickcr/eslint-plugin-react#159]
This line:

    let {adDetails, selectedIndex} = this.state,

is throwing the following warnings:

    !  405:11  'adDetails' is missing in props validation      react/prop-types
    !  405:27  'selectedIndex' is missing in props validation  react/prop-types

but these are attributes of `state`, not `props`!

**Comment by yannickcr** [comment]
Ok, I did reproduce your problem with the following code:

```jsx
class Hello extends React.Component {
  render() {
    var {adDetails, selectedIndex} = this.state, something = this.props;
    return <div>{adDetails}</div>;
  }
}
```

The problem seems to occur only if you are using `this.props` in a multiple variables declaration.

I'll fix that.

**Comment by evandavis** [comment]
Has this been updated on NPM? I'm still getting the error.

**Comment by yannickcr** [comment]
No, the fix is not published on npm yet. I plan to publish it this evening.
